mocha-multi.js

```diff
@@ -71,7 +71,7 @@ function convertSetup(reporters) {
       try {
         setup = setup.concat(convertSetup(JSON.parse(fs.readFileSync(reporters[reporter]))));
       } catch (e) {
-        bombOut('bad_file', reporters[reporter], e);
+        bombOut('bad_file', reporters[reporter], e.message);
```

Comment:
This change seems unrelated to the reported issue - could you explain why it's included?

Reply:
This modification is related to an internal change in Node that has happened between v10.15.3 and v12.7.0.

It seems that displaying 'e' dumps a complete stack trace with the later version, which was not the case before, and this makes the tests fail.

This line goes with the 'Error: ' modification, using only the message part which seems to be what the tests expects.

Indeed it is not directly related to the 'stats' modification, but this was required for the CI part and to run the unit-tests

Reply:
Aha, that makes sense - thanks.